### ISSUE TITLE
select_performance improvements

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#1837]
It's about performance so the number of selected fields is what makes the difference. I use 10 as a threshold but maybe it's a setting. So if either the table or the corresponding structure have more then 10 fields, it should raise an issue.

_Originally posted by @mbtools in https://github.com/abaplint/abaplint/issues/1834#issuecomment-794367227_

**Comment by larshp** [comment]
moving to https://github.com/abaplint/abaplint/issues/1945

**Comment by larshp** [comment]
@mbtools parameter added, see https://rules.abaplint.org/select_performance/

**Comment by mbtools** [comment]
I tried the parameter but I don't think it will work for tables that are not in the repo. Example:

```abap
DATA gt_cvers TYPE TABLE OF cvers.
SELECT * FROM cvers INTO TABLE gt_cvers. "cvers has 4 fields
```

not sure if we can solve that

**Comment by larshp** [comment]
the tables(in this case CVERS) must be known to the linter, you can add CVERS in a dependency repo, and it will be taken into account
